src/common/image.c

```diff
@@ -1792,10 +1793,19 @@ void dt_image_refresh_makermodel(dt_image_t *img)
   if(!img->camera_maker[0] || !img->camera_model[0] || !img->camera_alias[0])
   {
     // We need to use the exif values, so let's get rawspeed to munge them
-    dt_rawspeed_lookup_makermodel(img->exif_maker, img->exif_model,
-                                  img->camera_maker, sizeof(img->camera_maker),
-                                  img->camera_model, sizeof(img->camera_model),
-                                  img->camera_alias, sizeof(img->camera_alias));
+    int found = dt_rawspeed_lookup_makermodel(img->exif_maker, img->exif_model,
+                                              img->camera_maker, sizeof(img->camera_maker),
+                                              img->camera_model, sizeof(img->camera_model),
+                                              img->camera_alias, sizeof(img->camera_alias));
+
+    if(found == FALSE)
```

Comment:
@TurboGit rawspeed ships a  xml config for all supported cameras and to return proper names for make/model.
But we can not use these logic. For files decoded via libraw, a separate logic is needed to map make/model.

Reply:
Solved with a exiv->clean model mapping.